src/main/java/mate/academy/spring/controller/OrderController.java

```
@@ -34,15 +34,16 @@ public OrderController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/complete")
-    public OrderResponseDto completeOrder(@RequestParam Long userId) {
-        ShoppingCart cart = shoppingCartService.getByUser(userService.get(userId));
+    public OrderResponseDto completeOrder(Authentication authentication) {
+        ShoppingCart cart = shoppingCartService
+                .getByUser(userService.findByEmail(authentication.getName()).get());
```

it would be better to throw an informative exception if we can't find user